README.md

```diff
@@ -399,6 +399,8 @@ to Miri failing to detect cases of undefined behavior in a program.
 * `-Zmiri-track-weak-memory-loads` shows a backtrace when weak memory emulation returns an outdated
   value from a load. This can help diagnose problems that disappear under
   `-Zmiri-disable-weak-memory-emulation`.
+* `-Zmiri-force-page-size=<num>` overrides the default page size for an architecture, in multiples of 1k.
+  `4` is default for most targets. This value should always be a power of 2 and nonzero.
```

Comment:
What is the motivation for this flag?

I don't have fundamental objections here, but if we have such a flag we definitely need a test for it, too.

Reply:
Mostly because Linux allows variable page sizes (Asahi Linux is an important example I think) and some other platforms also have non-4k pages e.g. Windows on Alpha and IA64 (https://devblogs.microsoft.com/oldnewthing/20210510-00/?p=105200). I'll add a test, though